chat/models.py

```diff
@@ -0,0 +1,53 @@
+from django.conf import settings
+from django.db import models
+from django.dispatch import receiver
+
+
+def _get_service_choices():
+    return [(p, p) for p in settings.CHAT_SERVICES.keys()]
+
+
+class ChatRoom(models.Model):
```

Comment:
Can you add some comments specifying what this is? i.e. it can include both text and voice etc.

Reply:
Done!